src/App.jsx

```diff
@@ -1,19 +1,26 @@
 import React from "react";
 import Box from "@material-ui/core/Box";
+import { IntlProvider } from "react-intl";
 import Router from "./router/Router";
+import useLocale from "./hooks/useLocale";
 
-const App = () => (
-  <Box
-    mx="auto"
-    display="flex"
-    flexDirection="column"
-    alignItems="center"
-    textAlign="center"
-  >
-    <div className="App" data-test="component-app">
-      <Router data-test="router" />
-    </div>
-  </Box>
-);
+const App = () => {
+  const { locale, messages } = useLocale();
 
+  return (
+    <IntlProvider locale={locale} messages={messages}>
```

Comment:
💡 You could include `en-US` as the default language here. If you do, maybe we should create a `constants.js` file in the `utils` directory, set the default up there, and reference it here.

Reply:
Addressed:  45ec7fdea5f6bd94f02f1ae272420aa24a00736c